pkg/tagger/tagger.go

```diff
@@ -289,6 +318,7 @@ func (t *Tagger) List(highCard bool) response.TaggerListResponse {
 // contents to a new slice. As strings are references, the size of
 // the new array is small enough.
 func copyArray(source []string) []string {
+	sort.Strings(source) // TODO I'm not sure
```

Comment:
The `cachedAll` slice contains the low card tags first, then the high card. This allows the `cachedLow` slice to share the same backing array. We should not sort this array, or we'll mess up the low card tags.